packages/devtools_app/lib/src/screens/performance/panes/raster_stats/raster_stats_controller.dart

```diff
@@ -68,4 +68,14 @@ class RasterStatsController extends PerformanceFeatureController {
   void clearData() {
     setData(null);
   }
+
+  @override
+  Future<void> init() {
+    return Future.value();
+  }
+
+  @override
+  Future<void> onBecomingActive() {
+    return Future.value();
+  }
```

Comment:
what lint is forcing these overrides?

Reply:
if `PerformanceFeatureController` returns these values by default, do the subclasses need these empty overrides?

Reply:
This was a judgment call. The base class had 3 methods tagged as async that weren't really async. 1 of the 3 methods was actually overridden everywhere so could be made abstract. These other two were overridden almost everywhere except for this case where a no-op implementation had to be added. In general, if a member should almost always be overridden I think it is cleaner to make it abstract. In addition, overriding a method returning future is very ugly as you end up awaiting the super version of the method which is quite silly given the super version was. a no-op.